tracking/lib/presentation/utils/widget_util.dart

```diff
@@ -13,19 +12,21 @@ class WidgetUtil {
   //   _loadingOverlayEntry = showOverlay((_, __) => const DSLoadingIndicator(),
   //       duration: const Duration(hours: 100));
   // }
+  final GlobalKey<NavigatorState> navigatorKey = GlobalKey<NavigatorState>();
+  NavigatorState? get navigatorState => navigatorKey.currentState!;
 
   void dismissGlobalLoadingOverlay() {
     _loadingOverlayEntry?.dismiss(animate: false);
   }
 
-  void closeGlobalKeyboard({BuildContext? context}) {
-    if (context != null) {
-      FocusScope.of(context).unfocus();
-      return;
-    }
+  void closeGlobalKeyboard({required BuildContext context}) {
     FocusScope.of(navigatorKey.currentContext!).requestFocus(FocusNode());
```

Comment:
`FocusScope.of(context).unfocus();`
gọi thằng này chứ sao vẫn dùng `navigatorKey` thế kia... có context truyền vào rồi mà